numpy/core/tests/test_deprecations.py

```diff
@@ -190,10 +190,10 @@ def test_string(self):
         b = np.array(['a', 'b', 'c'])
         assert_raises(ValueError, lambda x, y: x == y, a, b)
 
-        # The empty list is not cast to string, as this is only to document
-        # that fact (it likely should be changed). This means that the
-        # following works (and returns False) due to dtype mismatch:
-        a == []
+        # The empty list is not cast to string, and this used to pass due
```

Comment:
More accurate would be "The empty list is cast to float64", I think?

Reply:
True, but not quite worth rerunning the whole test suite...